pom.xml

```diff
@@ -72,7 +72,7 @@
 		<module>spring-cloud-dataflow-registry</module>
 		<module>spring-cloud-dataflow-rest-resource</module>
 		<module>spring-cloud-dataflow-server-core</module>
-		<module>spring-cloud-dataflow-server-local-autoconfig</module>
+		<module>spring-cloud-dataflow-autoconfigure</module>
```

Comment:
Can we make this as `spring-cloud-dataflow-server-autoconfigure`?

Reply:
Point is that my hope is that eventually, maybe somewhere in 2.1.x line we have one autoconfigure jar instead of 7 we use now.

Reply:
We need to stop doing this:
```
$ grep EnableAutoConfiguration */src/main/resources/META-INF/spring.factories
spring-cloud-dataflow-configuration-metadata/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration:\
spring-cloud-dataflow-platform-cloudfoundry/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
spring-cloud-dataflow-platform-kubernetes/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
spring-cloud-dataflow-rest-client/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
spring-cloud-dataflow-server-core/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
spring-cloud-dataflow-server-local-autoconfig/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
spring-cloud-dataflow-shell-core/src/main/resources/META-INF/spring.factories:org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```
Every separate jar makes app startup slower.